site/downloads.html

```diff
@@ -207,7 +207,7 @@ <h5>Latest News</h5>
 });
 </script>
 
-<h2 id="download-apache-sparktrade">Download Apache Spark&#8482;</h2>
+<h2 id="download-apache-spark">Download Apache Spark&#8482;</h2>
```

Comment:
This one seems due to a rule change. I checked this id is uniquely used once here.